textalyze.js

```diff
@@ -14,15 +14,28 @@ function itemCounts(array) {
   return counts;
 }
 
-module.exports = { itemCounts };
+/**
+ * Given an input String, returns an Array containing each letter
+ * @param {String} string - The String to convert to array
+ * @param {Array} array - The Array containing the characters from the string
```

Comment:
I made this mistake in my code, too, but this should be `@returns` and not `@param`, since it describes what the function returns.

Reply:
Thanks, I didn't notice it, just the bad copy and paste behavior